root/edit/details/EditPlace.js

```diff
@@ -0,0 +1,134 @@
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import DescriptiveLink from
+  '../../static/scripts/common/components/DescriptiveLink';
+import Diff from '../../static/scripts/edit/components/edit/Diff';
+import FullChangeDiff from
+  '../../static/scripts/edit/components/edit/FullChangeDiff';
+import {formatCoordinates} from '../../utility/coordinates';
+import formatDate from '../../static/scripts/common/utility/formatDate';
+import yesNo from '../../static/scripts/common/utility/yesNo';
+
+type PlaceEditT = {
+  ...EditT,
+  +display_data: {
+    +address?: CompT<string>,
+    +area?: CompT<AreaT | null>,
+    +begin_date?: CompT<PartialDateT>,
+    +comment?: CompT<string>,
+    +coordinates?: CompT<CoordinatesT | null>,
+    +end_date?: CompT<PartialDateT>,
+    +ended?: CompT<boolean>,
+    +name: CompT<string>,
+    +place: PlaceT,
+    +type?: CompT<PlaceTypeT | null>,
+  },
+};
+
+const EditPlace = ({edit}: {edit: PlaceEditT}) => {
+  const display = edit.display_data;
+  const place = display.place;
+  const address = display.address;
+  const area = display.area;
+  const coordinates = display.coordinates;
+  const beginDate = display.begin_date;
+  const endDate = display.end_date;
+  const ended = display.ended;
+  const oldTypeName = display.type?.old?.name ?? '';
+  const newTypeName = display.type?.new?.name ?? '';
+  return (
+    <table className="details edit-place">
+      <tbody>
+        <tr>
+          <th>{addColonText(l('Place'))}</th>
+          <td><DescriptiveLink entity={place} /></td>
+        </tr>
+        {display.name ? (
+          <Diff
+            label={addColonText(l('Name'))}
+            newText={display.name.new}
+            oldText={display.name.old}
+            split="\s+"
+          />
+        ) : null}
+        {display.comment ? (
+          <Diff
+            label={addColonText(l('Disambiguation'))}
+            newText={display.comment ? display.comment.new : ''}
+            oldText={display.comment ? display.comment.old : ''}
+            split="\s+"
+          />
+        ) : null}
+        {display.type ? (
+          <FullChangeDiff
+            label={addColonText(l('Type'))}
+            newContent={newTypeName ? lp_attributes(newTypeName,
+                                                    'place_type') : ''}
+            oldContent={oldTypeName ? lp_attributes(oldTypeName,
+                                                    'place_type') : ''}
+          />
+        ) : null}
+        {address ? (
+          <Diff
+            label={addColonText(l('Address'))}
+            newText={address.new ? address.new : ''}
+            oldText={address.old ? address.old : ''}
+            split="\s+"
+          />
+        ) : null}
+        {area && area.new?.gid === area.old?.gid ? (
```

Comment:
Pretty sure you want a diff if they're NOT the same, here :)

Reply:
taken care of it,  thanks :)